packages/react-i18n/src/test/i18n.test.ts

```diff
@@ -1065,6 +1066,92 @@ describe('I18n', () => {
     });
   });
 
+  describe('#formatName()', () => {
+    it('returns only the firstName when lastName is missing', () => {
+      const i18n = new I18n(defaultTranslations, {locale: 'en'});
+
+      expect(i18n.formatName('first')).toStrictEqual('first');
+      expect(i18n.formatName('first', '')).toStrictEqual('first');
+    });
+
+    it('returns only the lastName when firstName is missing', () => {
+      const i18n = new I18n(defaultTranslations, {locale: 'en'});
+
+      expect(i18n.formatName('', 'last')).toStrictEqual('last');
+    });
+
+    it('defaults to firstName for unknown locale', () => {
+      const i18n = new I18n(defaultTranslations, {locale: 'unknown'});
+
+      expect(i18n.formatName('first', 'last')).toStrictEqual('first');
+    });
+
+    it('uses fallback locale value for unknown locale', () => {
+      const i18n = new I18n(defaultTranslations, {locale: 'fr-CA'});
```

Comment:
`fr-CA` is not an 'unknown' locale?

Reply:
That label wasn't explicit enough, it's now `uses fallback locale for locale without custom formatter`.

Reply:
I get it now!